tests/agent-templates.generations-post.test.ts

```
@@ -0,0 +1,297 @@
+
+const sampleBody = {
+  source: TEST_SOURCE,
+  inputs: { text: "build me a productivity assistant" },
+};
+
+async function cleanup() {
+  await prisma.agentTemplateGeneration.deleteMany({
+    where: { ownerAccountId: ADMIN_ACCOUNT_ID, source: TEST_SOURCE },
+  });
+  await prisma.agentTemplate.deleteMany({
+    where: { ownerAccountId: ADMIN_ACCOUNT_ID, agentName: fakeTemplate.agentName },
```

ESLint

[error] Replace `·ownerAccountId:·ADMIN_ACCOUNT_ID,·agentName:·fakeTemplate.agentName` with `⏎······ownerAccountId:·ADMIN_ACCOUNT_ID,⏎······agentName:·fakeTemplate.agentName,⏎···`

(prettier/prettier)